static/images/README.md

```diff
@@ -1,4 +1,19 @@
-inkscape -w 16 -h 16 -o 16.png logos/scipy.svg
-inkscape -w 32 -h 32 -o 32.png logos/scipy.svg
-inkscape -w 48 -h 48 -o 48.png logos/scipy.svg
+Add logo with name `logo.svg`.
```

Comment:
Looks like the main logo has a fixed filename, but the footer logo is still customizable?  Perhaps both or none shoul dbe.

Reply:
I think I got that one too, now.